package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "bitflux-openfin",
   "version": "10.0.0-rc.1",
+  "type": "develop",
```

Comment:
So to do a 'proper' release we need to manually change this line to `"type": "master"`? Is that correct?

If so, is there a way to automate that into the `grunt release(/deploy?)` task?